client/lib/sites-list/list.js

```diff
@@ -528,9 +573,9 @@ SitesList.prototype.getVisible = function() {
  *
  * @api public
  **/
-SitesList.prototype.getVisibleAndNotRecent = function() {
+SitesList.prototype.getVisibleAndNotRecentNorStarred = function() {
```

Comment:
This is a bit of a mouthful. Maybe pass through an options object, so we can return the sites we want:
```
SitesList.prototype.getSites = function ( options = { visible: true, favorites: false , recent: false} ) {
```